.eslintrc

```diff
@@ -1,3 +1,11 @@
 {
-  "extends": "standard"
+  "extends": "standard",
+  "parserOptions": {
+    "sourceType": "script"
```

Comment:
@develar `standard` always lints files as modules so that users don't need to specify which mode they're in. There's little downside to always running in module mode.